_posts/2021-12-10-dashboards-plugins-intro.markdown

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: "Introduction to OpenSearch Dashboard Plugins"
+authors:
+  - ashwinpc
+  - vemsarat
+date: 2021-12-08
+categories:
+  - technical-post
+twittercard:
+  description: "Plugins are fundamental to how Opensearch works, and that similarity extends to OpenSearch Dashboards too..."
+---
+
+Plugins are fundamental to how Opensearch works, and that similarity extends to OpenSearch Dashboards too. All major components and services used within Dashboards is a plugin. Similar to OpenSearch, additional functionality can also be added using external plugins. As a follow up the blog post on how plugins work for Opensearch, in this post we will explore how plugins work for OpenSearch Dashboards.
```

Comment:
that -> the
Not a grammatical expert, but some how it feels `the` would be better.

Reply:
"_All major components and services used within Dashboards is a plugin_" 
As a reader I don't know what these components and services are. It would be helpful to explain them.

The sentence starts with plural forms and converges with singular form. Probably something like `... Dashboards are plugins.`

Reply:
"_additional functionality can also be added using external plugins_"
Do we have internal and external plugins? Are they treated differently across the system?

Reply:
Thanks for the feedback Sarat, I'll incorporate this feedback with any others in the next revision.

Reply:
I've redone this paragraph to make it easier to understand while keeping it short so that i can focus on what a plugin is sooner.